platforms/core-configuration/model-reflect/build.gradle.kts

```diff
@@ -0,0 +1,25 @@
+plugins {
+    id("gradlebuild.distribution.api-java")
+}
+
+description = "Implementation of model reflection"
```

Comment:
This really seemed like a half-baked description to me. 

I find these project-level descriptions quite useful when navigating to a project I never have, as it is able to define the scope of the project:

1. What I can find in there
2. What _should not_ belong in there
3. When I should and should not depend on it

Two questions I have after reading this:

1. What is `model reflection`? 
2. Is there a main class/interface in here that acts an an entry-point to the functionalities implemented in this module?

Reply:
> This really seemed like a half-baked description to me.

Intentionally so. This was mainly a mechanical refactoring.

Reply:
Sorry to jump the gun if this project is still a moving target. 

I think it would be nice to revisit this description once the purpose of this module is more well-defined